### PR TITLE
Fix import in integration tests

### PR DESCRIPTION
**Pull request by pmrv**
Was pyiron instead of pyiron_atomistics

### CODE REVIEW

**Comment by pmrv** [comment]
@sudarsan-surendralal Can you see what's up in the notebook test?

**Comment by sudarsan-surendralal** [comment]
Sorry just saw this! Will look into this now

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
-  pull_request:
-    types: [ labeled ]
+  push:
+      paths:
+        - 'notebooks/integration'
 
 jobs:
   build:
-    if: ${{ github.event.label.name == 'integration' }}
+#    if: ${{ github.event.label.name == 'integration' }}
```
Comment (jan-janssen) [inline .github/workflows/integration.yml]: Should this be enabled again? @sudarsan-surendralal

File: pyiron_atomistics/atomistics/structure/atoms.py [hunk 2]
```diff
@@ -720,7 +720,7 @@ def get_chemical_indices(self):
             numpy.ndarray: A list of chemical indices
 
         """
-        return self.indices
+        return self.indices.copy()
```
Comment (jan-janssen) [inline pyiron_atomistics/atomistics/structure/atoms.py]: Is this related to this pull request?

**Comment by jan-janssen** [comment]
I have two small questions, apart from this - this pull request seems to be ready to be merged and should be merged sooner rather than later as other pull requests fail while this is not merged.

File: pyiron_atomistics/atomistics/structure/atoms.py (hunk 2 above)
Comment (sudarsan-surendralal) [reply pyiron_atomistics/atomistics/structure/atoms.py]: This is from https://github.com/pyiron/pyiron_atomistics/pull/350

**Comment by sudarsan-surendralal** [comment]
> 
> 
> I have two small questions, apart from this - this pull request seems to be ready to be merged and should be merged sooner rather than later as other pull requests fail while this is not merged.

I'm not sure that it is. If you notice, the integration tests still are not run on subsequent commits on this PR but only at the time of creation of this PR. So we don't really know if @pmrv's changes fixed the test. I think [this](https://github.com/pyiron/pyiron_atomistics/pull/352) PR must be merged before we merge this.

**Comment by pmrv** [comment]
@sudarsan-surendralal Is there anything left to do on this?

**Comment by sudarsan-surendralal** [comment]
> 
> 
> @sudarsan-surendralal Is there anything left to do on this?

We still don't know if these changes actually work since the integration notebooks still did not run even if we re-set the integration label.

**Comment by sudarsan-surendralal** [comment]
Sorry @pmrv my bad. I didn't realize that I had already merged your changes through [this PR](https://github.com/pyiron/pyiron_atomistics/pull/353). And with further changes to the notebook (https://github.com/pyiron/pyiron_atomistics/pull/366), the integration tests now work. IMO we should close this PR.
